client/post-editor/editor-ground-control/index.jsx

```diff
@@ -26,7 +26,7 @@ const Card = require( 'components/card' ),
 import { setDate } from 'state/ui/editor/post/actions';
 
 function isPostEmpty( props ) {
-	return ( props.isNew && ! props.isDirty ) || ! props.hasContent;
+	return ! props.hasContent || ( ! props.isNew && ! props.isDirty );
```

Comment:
Hrm. new logic still isn't right. Share a post, publish is available. Then auto-save kicks in and publish goes away.